app/kumactl/cmd/install/testdata/install-tracing.defaults.golden.yaml

```diff
@@ -0,0 +1,156 @@
+
+---
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: kuma-tracing
+---
+apiVersion: v1
+kind: List
+items:
+  - apiVersion: apps/v1
+    kind: Deployment
+    metadata:
+      name: jaeger
+      namespace: kuma-tracing
+      labels:
+        app: jaeger
+        app.kubernetes.io/name: jaeger
+        app.kubernetes.io/component: all-in-one
+    spec:
+      selector:
+        matchLabels:
+          app: jaeger
+      replicas: 1
+      strategy:
+        type: Recreate
+      template:
+        metadata:
+          labels:
+            app: jaeger
+            app.kubernetes.io/name: jaeger
+            app.kubernetes.io/component: all-in-one
+          annotations:
+            prometheus.io/scrape: "true"
+            prometheus.io/port: "16686"
+        spec:
+          containers:
+            -   env:
+                  - name: COLLECTOR_ZIPKIN_HTTP_PORT
+                    value: "9411"
+                image: jaegertracing/all-in-one
```

Comment:
Is it a good idea to leave here the latest version? What if Jaeger releases a new version and this YAML is not compatible with it? I think it's more reliable to lock the version here. Of course this comes at the cost of updating the version every now and then, but then we are sure that we are providing the command that works.

We locked the version in metrics Prometheus + Grafana deployment.

Reply:
Done.